src/core/Tile/Tile.tsx

```diff
@@ -223,7 +225,7 @@ export const Tile = (props: TileProps) => {
             <IconButton
               styleType='borderless'
               size='small'
-              className={cx('iui-more-options', {
+              className={cx('iui-tile-more-options', {
                 'iui-visible': isMenuVisible,
               })}
               aria-label='More options'
```

Comment:
Was not allowed to comment there, but there is no `iui-tile-button` class

Reply:
Huh. I see the CSS PR didn't even have it before. Why did we add it to React then?

Reply:
I think that is a really good question

Reply:
Found it :)

https://github.com/iTwin/iTwinUI/commit/ebcc9b28df136ab6c66e29b388b252ae3d924370#diff-fde6de9322de8edfd8f91b47b9dc6e00ca10ba514a532e2d73a61c978e3f99b3